### PR TITLE
Adjusted item translations (german)

### PR DESCRIPTION
**Pull request by marcelbrode**
Hello, first time contribution here.
I hope everythign is according the your guide lines.

Hope this helps :)

### CODE REVIEW

File: EN/ItemName_EN.txt [hunk 1]
```diff
@@ -919,7 +919,7 @@ ItemName_EN = {
     ItemName_Base.JamMarmalade = "Marmalade",
     ItemName_Base.JamesPic = "Picture of James Garcia",
     ItemName_Base.JarLid = "Jar Lid",
-    ItemName_Base.JellyBeans = "Jellybeans",
+    ItemName_Base.JellyBeans = "Jelly beans",
```
Comment (Faalagorn) [inline EN/ItemName_EN.txt:922]: It would be Jelly Beans if we'd follow the naming convention

File: EN/ItemName_EN.txt [hunk 2]
```diff
@@ -1652,8 +1652,8 @@ ItemName_EN = {
     ItemName_Base.Tie_Worn = "Clip-on Tie",
     ItemName_Base.Tie_Worn_Spiffo = "Clip-on Spiffo Tie",
     ItemName_Base.TightsBlack = "Tights",
-    ItemName_Base.TightsBlackSemiTrans = "Tights Semi Transparent",
-    ItemName_Base.TightsBlackTrans = "Tights Transparent",
+    ItemName_Base.TightsBlackSemiTrans = "Semi-Transparent Tights",
```
Comment (Faalagorn) [inline EN/ItemName_EN.txt:1655]: I addressed that in #939 though without a hyphen, however, if you add hyphen, currently the style is that the second word after it is set to be lowercase

File: EN/ItemName_EN.txt [hunk 3]
```diff
@@ -1919,12 +1919,12 @@ ItemName_EN = {
     ItemName_Radio.TvAntique = "Antique Television",
     ItemName_Radio.TvBlack = "ValuTech Television",
     ItemName_Radio.TvWideScreen = "Premium Technologies Television",
-    ItemName_Radio.WalkieTalkie1 = "Toys-R-Mine Walkie Talkie",
-    ItemName_Radio.WalkieTalkie2 = "ValuTech Walkie Talkie",
-    ItemName_Radio.WalkieTalkie3 = "Premium Technologies Walkie Talkie",
-    ItemName_Radio.WalkieTalkie4 = "Tactical Walkie Talkie",
-    ItemName_Radio.WalkieTalkie5 = "US Army Walkie Talkie",
-    ItemName_Radio.WalkieTalkieMakeShift = "Makeshift Walkie Talkie",
+    ItemName_Radio.WalkieTalkie1 = "Toys-R-Mine Walkie-Talkie",
```
Comment (Faalagorn) [inline EN/ItemName_EN.txt:1922]: Currently it's without dash in moveables, so I'd leave it as is.

File: EN/ItemName_EN.txt (hunk 1 above)
Comment (marcelbrode) [reply EN/ItemName_EN.txt:922]: You're right, my bad!

File: EN/ItemName_EN.txt (hunk 2 above)
Comment (marcelbrode) [reply EN/ItemName_EN.txt:1655]: Didn't notice that. Good point

File: EN/ItemName_EN.txt (hunk 3 above)
Comment (marcelbrode) [reply EN/ItemName_EN.txt:1922]: According to Merriam Webster, the dash is the correct way, so I'd rather correct it in Movables as well

File: EN/ItemName_EN.txt [hunk 4]
```diff
@@ -1557,9 +1557,9 @@ ItemName_EN = {
     ItemName_Base.SmokeBombSensorV2 = "Smoke Bomb with Sensor",
     ItemName_Base.SmokeBombSensorV3 = "Smoke Bomb with Sensor",
     ItemName_Base.SmokeBombTriggered = "Smoke Bomb with Timer",
-    ItemName_Base.Smore = "Smore",
+    ItemName_Base.Smore = "S'More",
```
[the remark this answers is not in the document]
Comment (marcelbrode) [reply EN/ItemName_EN.txt:1560]: Thanks!

**Comment by nsKpz** [comment]
DE Änderungen werden in späteren Commit eingebunden.

Die EN Änderungen nicht, da wir unveröffentlichte Änderungen im Hintergrund haben.

**Pull request closed without merging**